client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/test/payment-method-add.jsx

```diff
@@ -3,24 +3,30 @@
  */
 
 import { render, act } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from 'react-query';
 import { Provider } from 'react-redux';
 import { createStore } from 'redux';
 import PaymentMethodAdd from '../index';
 
 describe( '<PaymentMethodAdd>', () => {
 	test( 'should render correctly and match the snapshot', async () => {
 		const promise = Promise.resolve();
+		const queryClient = new QueryClient();
 		const initialState = {
 			ui: { section: 'test' },
 			documentHead: { unreadCount: 1 },
+			sites: { items: {} },
+			currentUser: { capabilities: {} },
 		};
 
 		const store = createStore( ( state ) => state, initialState );
 
 		const { container, getByText } = render(
-			<Provider store={ store }>
-				<PaymentMethodAdd />
-			</Provider>
+			<QueryClientProvider client={ queryClient }>
+				<Provider store={ store }>
+					<PaymentMethodAdd />
+				</Provider>
+			</QueryClientProvider>
```

Comment:
Now that licenses can be issued from the this component, these mocks were needed for testing.